server/models/auth/updateUser.js

```diff
@@ -39,6 +45,18 @@ function updateUser({
       query += `mobile=?,`
       arr.push(mobile)
     }
+    if (department) {
```

Comment:
What if department is 0 ?

Reply:
I think it would better to use
` if ( Number.isInteger(department) ) {`

Reply:
didn't think of that, my bad :sweat_smile: